app/ui/view_x.c

```diff
@@ -531,7 +531,7 @@ void run_ux_review_flow(review_type_e reviewType, const ux_flow_step_t *const st
         case REVIEW_TXN:
         default:
 #ifdef APP_BLINDSIGN_MODE_ENABLED
-            if (app_mode_blindsign()) {
+            if (app_mode_blindsign_required()) {
```

Comment:
Should we check if app blind signing mode is enabled, and if it is required? Or are we just covering both at the same time?

Reply:
If you check both, we run into the following: BS is enabled but not required,  if we check if app_mode_blindsign is enabled we will force the required to true but indeed we shouldn't

Reply:
mmmm what a problem... because if we set required to true, even if the blind signing mode is not enabled, we will display the screens... right?

Reply:
There is no way to set the required to true on the code.
We can only set it to zero with app_mode_blindising_skip_ui(), otherwise it will follow app_mode_blindsign enabled or not value

Reply:
Initial possabilities:
BS is enabled by user required will be set to true;
BS is disabled by user required will be set to false;

From that point:
Code checks if BS is true -> Bs is true -> required set to true
Code checks if BS is false -> Bs is false -> required maintains its value. But if BS is false it was set by the user and at that point required will also be set to zero

Reply:
Amazing!